doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-debugging-tips.rst

```diff
@@ -3,71 +3,75 @@
 Analyzing and Debugging A Teuthology Job
 ========================================
 
-For scheduling an integration test please refer to, `Scheduling Test Run`_.
+To learn more about how to schedule an integration test, refer to `Scheduling
+Test Run`_.
 
-Once a teuthology run is successfully completed, we can access the results using
-pulpito dashboard, which looks like:
+When a teuthology run has been completed successfully, use pulpito dasboard to
+view the results::
 
-http://pulpito.front.sepia.ceph.com/<job-name>/<job-id>/
+   http://pulpito.front.sepia.ceph.com/<job-name>/<job-id>/
 
-or via sshing into teuthology server::
+or ssh into the teuthology server::
 
     ssh <username>@teuthology.front.sepia.ceph.com
 
-and accessing `teuthology archives`_, for example::
-  
-  nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+and access `teuthology archives`_, like this for example:
 
-.. note:: This would require Sepia lab access. To know how to request it, see:
+  .. prompt:: bash $
+
+     nano /a/teuthology-2021-01-06_07:01:02-rados-master-distro-basic-smithi/
+
+.. note:: This requires you to have access to the Sepia lab. To learn how to
+          request access to the Speia lab, see: 
           https://ceph.github.io/sepia/adding_users/
 
-On pulpito, the jobs in red specify either a failed or dead job.
-Here, a job is combination of daemons and configurations that are formed using
+On pulpito, the jobs in red specify either a failed job or a dead job.
+A job is combination of daemons and configurations that are formed using
 `qa/suites`_ yaml fragments.
-Taking these configurations, teuthology runs the tasks that are present in
+Teuthology uses these configurations and runs the tasks that are present in
 `qa/tasks`_, which are commands used for setting up the test environment and
 testing Ceph's components.
-These tasks help us in covering large subset of usecase scenarios and hence
-exposing the bugs which were uncaught by `make check`_ testing.
+These tasks cover a large subset of usecase scenarios and help to  
+expose the bugs that were not caught by `make check`_ testing.
 
 .. _make check: ../tests-integration-testing-teuthology-intro/#make-check
 
-A job failure hence might be because of:
+A job failure might be caused by one or more of the following:
 
 * environment setup(`testing on varied systems<https://github.com/ceph/ceph/tree/master/qa/distros/supported>_`):
   testing compatibility with stable realeases for supported versions.
 
 * permutation of config values: for instance, qa/suites/rados/thrash ensures to
-  test Ceph under stressful workload, so that we be able to catch corner case
+  test Ceph under stressful workload, so that we are able to catch corner-case
```

Comment:
I've altered this, but I think that it can probably be a little clearer. I'm making this note so that I have the memory of doing so. That way, I'll remember more vividly that I should rewrite this when I return to this later in the week to rewrite it.